frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java

```diff
@@ -127,7 +129,22 @@ private void handleListModels(ChannelHandlerContext ctx, FullHttpRequest req) {
     }
 
     private void handlePing(ChannelHandlerContext ctx) {
-        NettyUtils.sendJsonResponse(ctx, new StatusResponse("healthy"));
+        Map<String, String> status = new HashMap<>();
+        Map<String, Model> models = ModelManager.getInstance().getModels();
```

Comment:
This list may become large. It's better we only return overall status.
We have list models and describe model api, user can use that the retrieve each models status.

Reply:
It does seem a lot. Question is do we foresee handling 1000's of models at once. If not, this should be OK I think. In any case, I can change this and wait on feedback from customers on if this is useful.
For now I will return one of the two error codes 200 (OK) and 206 (Partial content) to distinguish between all the workers are running and only some of the workers are running, in case of SUCCESS. I will make that change.